src/JunctionTreeTypes.jl

```diff
@@ -84,8 +72,8 @@ mutable struct CliqStateMachineContainer{BTND, G <: AbstractDFG, InMemG <: InMem
   cliqSubFg::InMemG
   tree::BT
   cliq::TreeClique
-  parentCliq::Vector{TreeClique} #TODO deprecate
-  childCliqs::Vector{TreeClique} #TODO deprecate
+  # parentCliq::Vector{TreeClique} #TODO deprecate
+  # childCliqs::Vector{TreeClique} #TODO deprecate
```

Comment:
I think this is fine for now, but would at some point like to come back and evaluate if it's better in `csmc` to only keep links to `parent` and `children` rather than the entire tree.  That means we would rather do things like:
```julia
getParent(csmc::CliqStateMachineContainer) = csmc.parent
```

This might be important with large distributed trees?

Reply:
xref #990

Reply:
Yes, I just wanted to start with a clean slate. Its more to make sure that there is no dead code lingering around.

Reply:
good plan!